README.md

```diff
@@ -573,7 +573,7 @@ variable NOMAD_ADDR to include the IP address of your development
 machine. This can be done with:
 
 ```bash
-source common.sh && export NOMAD_ADDR=http://$(get_ip_address):4646
+source scripts/common.sh && export NOMAD_ADDR=http://$(get_ip_address):4646
```

Comment:
Maybe it should be ```./scripts/common.sh```

Reply:
Sure, let's change it. I'm not sure which one is more appropriate, the one above also worked.

```suggestion
source ./scripts/common.sh && export NOMAD_ADDR=http://$(get_ip_address):4646
```